scripts/src/main/resources/variables.bat

```diff
@@ -13,18 +13,14 @@ set MAIN_BRANCH=main
 set SCRIPTS_PATH=scripts
 set CONF_PATH=conf
 set WORKSPACES_PATH=workspaces
+set WORKSPACESVS_PATH=workspaces_vs
 rem set SETTINGS_REL_PATH=settings
 set SETTINGS_REL_PATH=development\settings
+set ECLIPSE_TEMPLATES_REL_PATH=eclipse\workspace
```

Comment:
I removed this flexibility because it does not appear to make any sense to change things like this. KISS.